bin/kano-post-clean

```diff
@@ -0,0 +1,30 @@
+#!/bin/sh
+#
+# kano-post-clean
+#
+# Copyright (C) 2018 Kano Computing Ltd.
+# License: http://www.gnu.org/licenses/gpl-2.0.txt GNU GPL v2
+#
+# Creates a clean post install environment
+# by enabling autocleaning
+
+
+# Clean the retrieved .deb packages from the local repo
+# and delete all .deb files from /var/cache/apt/archives
+# to free up important space
+post_clean() {
+
+    # Ensure there's no lock enabled
+    sudo rm /var/cache/apt/archives/lock &&
```

Comment:
Why do you need to do that? It seems like the kind of thing that requires a very good reason.

Reply:
Because when the `DPkg::Post-Invoke` triggers, the `apt-get` manager is still running in the background (that's why it keeps a lock file in place). That means it doesn't like to run another instance of itself (the one that it will run basically the `apt-get clean`), which I am allowing by removing the lock.

The other solution is to never run this file and just do as I am describing here (inline the `apt.d`):
[Pull @2852](https://github.com/KanoComputing/peldins/pull/2852).

Reply:
Okay, I don't think you should just remove the lock. So 2825 is an alternative to this PR?

Reply:
No, it goes together; My other solution was on the comments of #2852, i.e. alternatively just invoke the cleaning process through `apt.d` trigger (repasting here0:

```
DPkg::Post-Invoke { "rm -f /var/cache/apt/archives/*.deb /var/cache/apt/archives/partial/*.deb /var/cache/apt/*.bin || true"; };
APT::Update::Post-Invoke { "rm -f /var/cache/apt/archives/*.deb /var/cache/apt/archives/partial/*.deb /var/cache/apt/*.bin || true"; };

Dir::Cache::pkgcache "";
Dir::Cache::srcpkgcache
```

With this solution no other file will be added to the `/usr/bin`.